### ISSUE TITLE
Case sensitive versus case insentive or case preserving file systems

### ISSUE THREAD

**Issue by dragotin** [owncloud/client#1348]
Different systems have different file systems that handle file names differently. One particular problem is the case sensitivity handling which can cause all kinds of problems when operating in a cross platform environment.

We need to track the problems down and find a solution. This bug tracks the changes to be done on the desktop sync clients. 

There is owncloud/core#4747 to track this problem on sever side.

**Comment by phil-davis** [comment]
This discussion is a bit old, so maybe there is somewhere more current that an ongoing discussion of file-naming issues is happening? Anyway, I will post here!
In a (mostly or all) Windows clients use case, the Windows clients are "dumb" (case-preserving only) because the typical Windows file systems are that way anyhow. But users can induce sync issues from the web-browser interface by making case-sensitive-conflicting file names there.
Perhaps it would be useful to be able to set the server to "dumb case-preserving mode" - a server setting "Use dumbed-down case-preserving file naming only".
Then Windows clients would be happy - they are already dumbed-down because they are Windows. And the web-browser client could understand when the server is "dumbed-down case-preserving" and thus simply refuse to accept uploads/creations of files or folders that would create case-sensitive conflicts.
*nix and other clients could also understand the server setting and only upload the first of a case-sensitive-conflicting set of files, and log errors about the others.
That would allow the server admin to side-step this issue (at their discretion), pushing it back to the (*nix) case-sensitive clients (of which there might be none or few in use).

**Comment by PVince81** [comment]
New ticket for the server-side discussion: https://github.com/owncloud/core/issues/17161

**Comment by count0-krsk** [comment]
So, is any decision for problem now? Or developers just "to hammer a bolt" on this?
We have linux server, windows clients and stupid users what rename Folder to FOLDER often and then ask: "Why your buggy cloud don't syncing anything?"

**Comment by michaelstingl** [comment]
@pmaier1 One day, we need a decision here… Related: https://github.com/owncloud/documentation/issues/2832

**Comment by guruz** [comment]
@count0-krsk Can you post this as a new bug (or search for duplicates).
Changing the case of a file is/was supported AFAIK.
(The users rename via windows client or web interface?)

**Comment by count0-krsk** [comment]
Of course, if it requred. I search before post and found this. And more complex thread with platform-dependent issues (michaelstingl show link before), where writen what no "gold bullet" for mac, win and linux for now. 
Clients have many folders with sharing from one user to many other partially (some dirs to one group, some to other users). Some users rename Folder locally to FOLDER, add some files to it. Other see Folder with no new files and add their own. Then I must go to their PC's and sync manually...

**Comment by guruz** [comment]
@count0-krsk Please create a new issue, also by mentioning the exact server and client versions you are using. Thanks
